cmake/projects/glfw/hunter.cmake

```diff
@@ -78,6 +78,17 @@ hunter_add_version(
     455c97ea01a1b78ae39a72d00d86eb1c455e0dca
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    glfw
+    VERSION
+    "3.3.0-p5"
```

Comment:
Should be `3.3.0-f9923e9-p0`.

I've added section to documentation:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#consistency